app/src/main/java/ch/epfl/sdp/blindly/localDB/UserConverter.kt

```diff
@@ -0,0 +1,51 @@
+package ch.epfl.sdp.blindly.localDB
+
+import androidx.room.TypeConverter
+import java.lang.StringBuilder
+
+/**
+ * Class that help the local database to store complex types (lists)
+ *
+ */
+class UserConverter {
+
+    @TypeConverter
+    fun fromLocationToString(loc: List<Double>): String {
+        return loc[0].toString() + "," + loc[1].toString()
+    }
+
+    @TypeConverter
+    fun fromStringToLocation(str: String): List<Double> {
+        val values = str.split(",")
+        return listOf(values[0].toDouble(), values[1].toDouble())
+    }
+
+    @TypeConverter
+    fun fromStringListToString(sl: List<String>): String {
+        val len = sl.size
+        val str = StringBuilder()
+        for(i in 0 until len) {
+            str.append(sl[i])
+            if (i != len - 1) {
```

Comment:
Not very important but you can use TextUtils.join for this kind of things. 

```join(CharSequence delimiter, Iterable tokens)```

Returns a string containing the tokens joined by delimiters.

Reply:
tried it but can't test it with it